linode/manager
# feat: [M3-6735] - VPC Edit drawer

## Description 📝
Add the ability to edit a VPC

## Preview 📷
https://github.com/linode/manager/assets/115299789/f84e0cb6-06cb-4d19-a771-a66e37a7cdbc

## How to test 🧪
```
yarn test VPCEditDrawer
```
```
yarn test VPCRow
```
Create a VPC via the API, click on the edit button and change the label/description. Check the table/drawer again to ensure it updated
